pkgs/development/python-modules/fuzzywuzzy/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, python, fetchFromGitHub, python-Levenshtein, pycodestyle, hypothesis, pytest }:
+
+buildPythonPackage rec {
+  pname = "fuzzywuzzy";
+  version = "0.16.0";
+
+  src = fetchFromGitHub {
+    owner = "seatgeek";
+    repo = "fuzzywuzzy";
+    rev = "e0f2b0d206d1b21c072ac68a0f08f613ac59d86c";
```

```suggestion
    rev = version;
```

Similar to https://github.com/NixOS/nixpkgs/pull/53261/files#r245488593

---

Done